File: app/src/main/java/com/skeleton/mvp/ui/onboarding/signup/SignUpView.java
package com.skeleton.mvp.ui.onboarding.signup;

import com.skeleton.mvp.ui.base.BaseView;

/**
 * Created by clicklabs on 11/04/18.
 */

public interface SignUpView extends BaseView {
    /**
     * Show email error.
     *
     * @param resId the res id
     */
    void showEmailError(int resId);

    /**
     * Show phone number error.
     *
     * @param resId the res id
     */
    void showPhoneNumberError(int resId);


    /**
     * On sign up success.
     *
     * @param message the message
     */
    void onSignUpSuccess(String message);

}

File: app/src/main/java/com/skeleton/mvp/ui/onboarding/signup/SignUpView.kt
package com.skeleton.mvp.ui.onboarding.signup;

import com.skeleton.mvp.ui.base.locationbase.BaseLocationView;


/**
 * Developer: Geetanjali Gupta
 */
interface SignUpView : BaseLocationView {
    /**
     * Show email error.
     *
     * @param resId the res id
     */
    fun showEmailError(resId: Int);

    /**
     * Show phone number error.
     *
     * @param resId the res id
     */
    fun showPhoneNumberError(resId: Int);


    /**
     * On sign up success.
     *
     * @param message the message
     */
    fun onSignUpSuccess(message: String);

}
